frontend/sass/_icons.scss

```diff
@@ -97,3 +97,16 @@ $twelfth-column: 0;
 .icon.icon-logs:hover:before {
   background-position-x: $fourth-column;
 }
+
+.icon.icon-gear {
+  &:before {
+    @include icon();
+    height: 1em;
+    background-position: $second-column $second-row;
+    content: 'Gear icon';
+  }
+
+  &:hover:before {
+    background-position-x: $fifth-column;
```

Comment:
why is this needed?

Reply:
that's for "swapping" out the black icon to the blue icon on mouse hover.

Reply:
ah! clever.